srcpkgs/cups-filters/template

```diff
@@ -1,47 +1,19 @@
 # Template file for 'cups-filters'
 pkgname=cups-filters
-version=1.28.17
-revision=3
+version=2.0.0
+revision=1
 build_style=gnu-configure
-configure_args="--disable-static --with-rcdir=no --enable-avahi
- --with-browseremoteprotocols=DNSSD,CUPS
- --with-test-font-path=/usr/share/fonts/TTF/DejaVuSans.ttf"
-hostmakedepends="ghostscript glib-devel mupdf-tools pkg-config poppler liblouis"
-makedepends="avahi-glib-libs-devel cups-devel fontconfig-devel liblouis-devel
- ghostscript-devel lcms2-devel libqpdf-devel poppler-cpp-devel libexif-devel"
-depends="bc ghostscript libcups-filters-${version}_${revision} poppler"
-checkdepends="dejavu-fonts-ttf"
-conf_files="/etc/cups/cups-browsed.conf"
+configure_args="--disable-static --enable-avahi"
+hostmakedepends="pkg-config ghostscript mupdf-tools"
+makedepends="cups-devel libppd-devel libcupsfilters-devel"
 short_desc="OpenPrinting CUPS Filters"
 maintainer="Mohammed Anas <triallax@tutanota.com>"
-license="GPL-2.0-or-later, LGPL-2.1-or-later, MIT"
+license="Apache-2.0"
 homepage="https://wiki.linuxfoundation.org/openprinting/cups-filters"
 changelog="https://github.com/OpenPrinting/cups-filters/releases"
 distfiles="https://github.com/OpenPrinting/cups-filters/releases/download/${version}/cups-filters-${version}.tar.xz"
-checksum=270a3752a960368aa99d431fb5d34f4039b2ac943c576d840612d1d8185c9bb9
-lib32disabled=yes
```

Comment:
I'm not quite sure why `lib32disabled=yes` was set here, is there any reason for it?

Reply:
Never mind, re-reading your comment again,  I think I now grok what you were saying. I set `lib32disabled=yes` for `cups-browsed` as well since it has a binary (`/usr/lib/cups/backend/implicitclass`) in `/usr/lib`.